linux_os/guide/system/permissions/permissions_local/group.yml

```diff
@@ -0,0 +1,12 @@
+documentation_complete: true
+
+title: |-
+    Verify Permissions on Important Files and
+    Directories Are Configured in /etc/permissions.local
+
```

Comment:
I don't have a strong opinion here, this group looks fine here.
But it looks like it could be in the `software-integrity` group too, along side `aide` and `rpm_verification`.

Reply:
This one is debatable I think. Per my understanding, the rules under then permissions_local  group (i.e. run_chkstat) are to ensure the file permissions are configured accordingly. The ones in software-integrity are to ensure the files has not been tempered with. I am still relatively new to this project so I may not know all the historic context. But I am fine with moving run_chkstat to software-integrity if needed. Please let me know.

Reply:
Thank you for your point of view.
I think it makes sense to keep this group here.
It can also be moved later if needed.